src/views/Player.vue

```diff
@@ -223,26 +212,17 @@ export default {
           .catch(err => console.log(err))
     },
 
-    updateJourney(journeyType, event, detail){
-      var obj = [
-        String(this.getCurrentDateTime()),
-        String(event),
-        String(detail)
-      ]
-
-      switch(journeyType) {
-        case "questionJourney":
-          this.questionJourney.push(obj)
-          break;
-        case "playbackJourney":
-          this.playbackJourney.push(obj)
-          break;
-        case "fullscreenJourney":
-          this.fullscreenJourney.push(obj)
-          break;
-        default:
-          console.log("invalid journey type")
-      }
+    updateJourney(event, details = {}) {
+        // handle the case when fullscreen has been clicked but Plyr has not
+        // yet loaded -> this.player.currentTime = NaN
+        var player_time = this.hasPlyrLoaded ? this.player.currentTime : 0
+
+        this.journey.push({
+            'event': event,
+            'details': details,
+            'system_time': String(this.getCurrentDateTime()),
+            'player_time': String(player_time)
+        })
```

Comment:
standard format for ALL events

Reply:
I also had a doubt when I made a change in my PR, will "event" here work properly? JavaScript reads the word "event" not as a variable, but as an event. Do you think we should avoid using the word event here?

Reply:
achcha, haan I'll change